services/api-server/src/simcore_service_api_server/models/schemas/solvers_filters.py

```diff
@@ -0,0 +1,27 @@
+from typing import Annotated
+
+from pydantic import BaseModel, ConfigDict, Field
+
+
+class SolversListFilters(BaseModel):
+    """Filters for listing solvers"""
+
+    solver_id: Annotated[
+        str | None,
+        Field(
+            description="Filter by solver ID pattern",
+            examples=["simcore/services/comp/itis/sleeper", "simcore/services/comp/*"],
```

Comment:
Q: why not stick with the examples in the model_config field? THis way they are also tested automatically. This here provide nothing and defined the, in a different way than all other models are defined

Reply:
I see your point but this is a special case.
These fields are used as query parameters and this is the only way to get the examples in the documentation
If you have a better way to do it, i am open to suggestions.
I will add some extra tests though.

Reply:
I see your point but this is a special case.
These fields are used as query parameters and this is the only way to get the examples in the documentation
If you have a better way to do it, i am open to suggestions.
I will add some extra tests though.

Reply:
I see your point but this is a special case.
These fields are used as query parameters and this is the only way to get the examples in the documentation
If you have a better way to do it, i am open to suggestions.
I will add some extra tests though.